tests/test_browser.py

```diff
@@ -3307,6 +3307,7 @@ def test_sdl2_mixer_music(self, formats, flags, music_name):
 
   @requires_graphics_hardware
   def test_cocos2d_hello(self):
+    self.emcc_args.remove('-Werror')
```

Comment:
Please add a comment why.

Also, could this just disable JS warnings specifically?